params/config.go

```diff
@@ -91,6 +91,26 @@ var (
 		},
 	}
 
+	// TestnetTrustedCheckpoint contains the light client trusted checkpoint for the Ropsten test network.
```

Comment:
This checkpoint is for the ethereum ropsten test network and I don't think we should include it.